backend/app/Models/BillingChange.php

```
@@ -0,0 +1,54 @@
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * Historic record of a user's license-count / MRR change.
+ *
+ * Rows are written by the app:refresh-analytics command whenever a user's billable
+ * usage (displays + boards×2) moves between snapshots. Inserts use DB::table() in
+ * that command; this model is for reading (admin panel, tests).
+ */
+class BillingChange extends Model
```

PHPStan (2.2.2)

[error] Class App\Models\BillingChange extends unknown class Illuminate\Database\Eloquent\Model.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)